syntax = "proto2";

package messaging;

message FindUser {
    required string packet_type = 1;
    optional string username = 2;
    optional bytes ipaddress = 3;
    optional string list_of_users = 4;
}
